### DISCUSSION TITLE
Handle deep linking of http/https URI's

### DISCUSSION THREAD

**Discussion by CliffAgius** [CommunityToolkit/Maui#1533] [New Feature Discussions]
the amazing @redth posted on X/Twitter that he created a sample for deep linking including a great github sample:

https://github.com/redth/maui.applinks.sample

I think we ask him nicely and look to bring this into the CT adding in the Win/Mac versions as well.

**Comment by GeorgeLeithead** [comment]
I saw the post by @Redth too.  As there is a lot of effort needed by the developer, I think that something like this integrated into the MCT and made 'easier' for the developer, would be of real benefit.

**Reply by bijington** [reply]
Is there much that we could do in the toolkit to make this process easier? From previous experience I've found a lot of the setup is specific to the app, unless I've been doing it wrong 😂

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers.
